test/fullItem.test.ts

```diff
@@ -313,7 +318,7 @@ describe("Module `fullItem`: fetches the item, data, and resources of an AGOL it
 
         const abc = {...MOCK_ITEM_PROTOTYPE};
         abc.type = "Group";
-        abc.item.id = "grp1234567890";
+        abc.item = {id: "grp1234567890"};
```

Comment:
This seems repeated a number of times... you could streamline this with Object.assign...
`const abc = Object.assign({}, MOCK_ITEM_PROTOTYPE, {type: "Group", item: {id: 'grp1234567890'}});`